Cython/Compiler/Code.py

```diff
@@ -336,28 +349,22 @@ def load(cls, util_code_name, from_file=None, **kwargs):
         if not from_file:
             utility_dir = get_utility_dir()
             prefix = util_code_name + '.'
-            try:
-                listing = os.listdir(utility_dir)
-            except OSError:
-                # XXX the code below assumes as 'zipimport.zipimporter' instance
-                # XXX should be easy to generalize, but too lazy right now to write it
-                import zipfile
-                global __loader__
-                loader = __loader__
-                archive = loader.archive
-                with closing(zipfile.ZipFile(archive)) as fileobj:
```

Comment:
Erm, well – where did this part go?

Reply:
I replaced the entire listdir() logic with checking a predefined set of file name extensions. 

This part is doing a listdir() on a zip file if I understood it correctly, thus it is not needed any more.

Reply:
Done via  #3280.